README.md

```diff
@@ -58,3 +55,4 @@ Feel free to check the issues page.
 
 📝 License
 This project is Open Source.
+.
```

Comment:
[OPTIONAL] Kindly consider adding a Get start section with the instruction on how to use the App/Project

Reply:
- [ ] Kindly always include an explanation, or a "why", for each requested change, this will make your comments more useful and clear

Reply:
[OPTIONAL] Kindly consider adding a Get start section with the instruction on how to use the App/Project. This is going to be useful for the people who check your projects and wants to give them a try!